src/components/VideoComponent.js

```diff
@@ -0,0 +1,66 @@
+/* Imports -------------------------------------------------------------------*/
+
+import Inferno from 'inferno';
+import Component from 'inferno-component';
+// import img from 'kalm-logo.png';
```

Comment:
Normally, importing images should be done through the html with an `<img>` tag or css with `background-image: url(...)`, unless there's some reason why you needed a reference in the JS